task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -0,0 +1,60 @@
+import time
+from threading import Thread
+
+from pyrsistent import m
+from six.moves.queue import Queue
+
+from task_processing.interfaces.task_executor import TaskExecutor
+
+
+class RetryingExecutor(TaskExecutor):
```

Comment:
i did consider runners, problem is they aren't composable. executors are due to very simple and strict interface.